Sources/XcbeautifyLib/Renderers/OutputRendering.swift

```diff
@@ -565,4 +565,80 @@ extension OutputRendering {
     func formatTestingStarted(group: TestingStartedCaptureGroup) -> String {
         colored ? group.wholeMessage.s.Bold.f.Cyan : group.wholeMessage
     }
+
+    func formatSwiftTestingRunStarted(group: SwiftTestingRunStartedCaptureGroup) -> String {
+        colored ? group.message.s.Bold : group.message
+    }
+
+    func formatSwiftTestingRunCompletion(group: SwiftTestingRunCompletionCaptureGroup) -> String {
+        return "Test run with \(group.numberOfTests) tests passed after \(group.totalTime) seconds"
+    }
+
+    func formatSwiftTestingRunFailed(group: SwiftTestingRunFailedCaptureGroup) -> String {
+        return "Test run with \(group.numberOfTests) tests failed after \(group.totalTime) seconds with \(group.numberOfIssues) issue(s)"
+    }
+
+    func formatSwiftTestingSuiteStarted(group: SwiftTestingSuiteStartedCaptureGroup) -> String {
+        return "Suite \(group.suiteName) started"
+    }
+
+    func formatSwiftTestingTestStarted(group: SwiftTestingTestStartedCaptureGroup) -> String? {
+        return nil
+    }
+
+    func formatSwiftTestingSuitePassed(group: SwiftTestingSuitePassedCaptureGroup) -> String {
+        return "Suite \(group.suiteName) passed after \(group.timeTaken) seconds"
+    }
+
+    func formatSwiftTestingSuiteFailed(group: SwiftTestingSuiteFailedCaptureGroup) -> String {
+        return "Suite \(group.suiteName) failed after \(group.timeTaken) seconds with \(group.numberOfIssues) issue(s)"
+    }
+
+    func formatSwiftTestingTestFailed(group: SwiftTestingTestFailedCaptureGroup) -> String {
+        let message = "\(group.testName) (\(group.timeTaken) seconds) \(group.numberOfIssues) issue(s)"
+        let wholeMessage = colored ? TestStatus.fail.foreground.Red + " " + message.f.Red : TestStatus.fail + " " + message
+        return Format.indent + wholeMessage
+    }
```

Comment:
Can you add an implementation for the GitHub Actions and TeamCity renderers? I think those one should annotate an error for each.